content/english/java-tictactoe/activity-5.md

```diff
@@ -3,5 +3,80 @@ title: "5. (Optional) AI Move"
 description: ""
 date: 2020-07-23T00:00:00Z
 weight: 6
-hidden: true
----
\ No newline at end of file
+hidden: false
```

Comment:
Can we add a pre-req here with "java - functions", and maybe a difficulty level as Intermediate/Advanced?

Reply:
Added 👍